### PR TITLE
CLI: more metadata for `get-aqs`

### PR DESCRIPTION
**Pull request by zmoon**
This adds a few more site metadata fields, including CBSA name, city name, state abbreviation, and EPA region.

cc: @btang1

### CODE REVIEW

**Comment by zmoon** [comment]
@btang1 sites where city name is "Not in a city" might be a good rural indicator.

File: melodies_monet/_cli.py [hunk 1]
```diff
@@ -1028,26 +1028,59 @@ def get_aqs(
...
+                    "State Code": "state_code",
+                    "State Name": "state_name",
+                    "State Abbreviation": "state_abbr",
+                    "County Code": "county_code",
+                    "County Name": "county_name",
+                    "EPA Region": "epa_region",  # note without R prefix
+                }
+            )
+        )
+        counties["epa_region"] = "R" + counties["epa_region"]
```
Comment (zmoon) [inline melodies_monet/_cli.py:1078]: AirNow EPA region number isn't zero-padded (R1 instead of R01, etc.). Should be consistent here.

Comment (zmoon) [reply melodies_monet/_cli.py]: Adressed in 5587b6f

**Comment by zmoon** [comment]
Current CI failures related to NCEI websites being down.

**Comment by zmoon** [comment]
@btang1 did you end up trying this out?

**Comment by btang1** [comment]
> @btang1 did you end up trying this out?

I will try it later today or tomorrow @zmoon

**Comment by zmoon** [comment]
Yeah this is on my fork's branch `cli-aqs-meta`.

**Comment by btang1** [comment]
> Yeah this is on my fork's branch `cli-aqs-meta`.

Is it in Monet? or Monet io?

**Comment by btang1** [comment]
> > Yeah this is on my fork's branch `cli-aqs-meta`.
> 
> Is it in Monet? or Monet io?

ok. I found it! it is in your melodies-monet. let me test it quick!

**Comment by btang1** [comment]
> Yeah this is on my fork's branch `cli-aqs-meta`.

Hi Zach, I tested and it works!!!
Screenshot 2024-10-09 at 1 42 40 PM


I can find new variables you mention in so2 data from AQS.

**Comment by btang1** [comment]
@zmoon

**Review by rschwant — APPROVED** [review]
Looks great, glad to have extra meta data available now!

**Pull request merged**